worker/src/RTC/RtpPacket.cpp

```diff
@@ -704,6 +711,10 @@ namespace RTC
 		packet->frameMarkingExtensionId      = this->frameMarkingExtensionId;
 		packet->ssrcAudioLevelExtensionId    = this->ssrcAudioLevelExtensionId;
 		packet->videoOrientationExtensionId  = this->videoOrientationExtensionId;
+		// Clone payload descriptor handler.
```

Comment:
This is not a clone. Is it safe by just copying it? It's not a pointer so this is ok right? And why didn't we do it before?

Reply:
It was a `unique_ptr` before, it is a `shared_ptr`. It's safe.

Today there is no place in the code where we need the `PayloadDescriptorHandler`  in cloned packets, but if we did it would fail. I consider this a fix.

Reply:
I'm double checking.

Reply:
Correct, we don't hold any reference in `PayloadDescirptorHandler` to the original buffer, we are OK.

Reply:
But comment should not say "clone" right?

Reply:
Very true. Comment fixed.